.travis/check_src_file_tag.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+return_code=0
+
+# Checking that source code has proper @file tag
+for file in `find * -regextype posix-extended -regex '.*\.(cpp|h|ih)'`
```

Comment:
Ok, a better (finds only files) and probably more universal (no regexp) command line would be:
```find * -type f \( -name \*.cpp -o -name \*.h -o -name \*.ih \)```